File: csharp/Concepts/Loops.cs
namespace ProgrammingFundamentals.Concepts
{
    public class Loops
    {
        public static void Demonstrate()
        {
            Console.WriteLine("\n=== Loops ===");

            // For loop
            Console.Write("Counting with for loop: ");
            for (int i = 1; i < 5; i++)
            {
                Console.Write($"{i} ");
            }
            Console.WriteLine();

            // While loop
            Console.Write("Counting with while loop: ");
            int count = 1;
            while (count < 5)
            {
                Console.Write($"{count} ");
                count++;
            }
            Console.WriteLine();
        }
    }
}

File: java/concepts/Loops.java
package concepts;

public class Loops {
    public static void demonstrate() {
        System.out.println("\n=== Loops ===");

        // For loop
        System.out.println("Counting with for loop:");
        for (int i = 1; i < 5; i++) {
            System.out.print(i + " ");
        }
        System.out.println(); // For a new line

        // While loop
        System.out.println("Counting with while loop:");
        int count = 1;
        while (count < 5) {
            System.out.print(count + " ");
            count++;
        }
        System.out.println();
    }
}
